test/integration/Admin.t.sol

```diff
@@ -2,10 +2,30 @@
 pragma solidity 0.8.26;
 
 import "test/BaseTest.sol";
+import {MockManager} from "test/mocks/MockManager.sol";
+import {CastLib} from "src/libraries/CastLib.sol";
 
 contract AdminTest is BaseTest {
+    using CastLib for *;
+
+    address[] threeMockAdapters;
+
     function setUp() public override {
         super.setUp();
+        adapter1 = new MockAdapter(address(gateway));
```

Comment:
Since you are using the `super.setUp()` and you inherit from the `BaseTest` you can use the `testAdapters`